<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBarang</class>
 <widget class="QWidget" name="FormBarang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>hapus</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="formGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="id_barangLabel">
      <property name="text">
       <string>id_barang</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="id_barangLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaBarangLabel">
      <property name="text">
       <string>nama barang</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaBarangLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="spesipikasiLabel">
      <property name="text">
       <string>spesipikasi</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="spesipikasiLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="hargaSatuanLabel">
      <property name="text">
       <string>harga satuan</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="hargaSatuanLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="simpan">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableBarang">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>381</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
